cfme/infrastructure/config_management/__init__.py

```diff
@@ -386,7 +361,7 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         """
         if not force and not self.exists:
             return
-        view = navigate_to(self, 'All')
+        view = navigate_to(self.parent, 'All')
         view.toolbar.view_selector.select('List View')
```

Comment:
Optional: As we are refactoring. I think better to move this to `resetter`.

Reply:
Agreed, moving